node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
```

Comment:
Don't allow an `else` statement if the `if` statement returns (unnecessary else). Makes it much easier to follow logical flow.
https://eslint.org/docs/rules/no-else-return
```js
// Old
function foo() {
  if (x) {
    return y;
  } else {
    return z;
  }
}
// New
function foo() {
  if (x) {
    return y;
  }
  return z;
}
```

Reply:
Is there a rule to enforce ternary here instead?

return x ? y : z

Reply:
I like it but it's hard to see how to fix the linter error at first glance